functions/Get-DbaConfiguration.ps1

```diff
@@ -0,0 +1,108 @@
+﻿FUNCTION Get-DbaConfiguration
```

Comment:
Get-DbaInstanceConfiguration be better?
Would indicate that these are instance level configs, rather than the database level configs that exist in 2016+